surveys/promise-rejections/survey.md

```diff
@@ -1,5 +1,70 @@
 # Node.js Promise reject use case survey
 
+Today, Node.js handles unhandled rejections by emitting a deprecation warning to stderr. The warning shows the stack where the rejection happened, and states that in future Node.js versions unhandled rejections will result on Node.js exitting with non-zero status code. We intend to remove the deprecation warning, replacing it with a stable behavior which might be different from the one described on the deprecation warning. We're running this survey to better understand how Node.js users are using Promises and how they are dealing with unhandled rejections today, so we can make an informed decision on how to move forward.
```

Comment:
Nit: 
> in future Node.js versions unhandled rejections will result on Node.js exitting with non-zero status code.

 in future Node.js versions unhandled rejections will result `in` Node.js `exiting` with non-zero status code.